### DISCUSSION TITLE
Ability to switch between projects with keyboard shortcut?

### DISCUSSION THREAD

**Discussion by Coteh** [Kong/insomnia#4665] [Help]
Hey all, just wondering if there's a way one can quickly jump between projects? Similar to how ctrl-p allows for quick switching between individual requests within a collection.

I have request collections grouped in multiple projects, and it would be nice if there was a way to quickly switch between them. Currently I need to click on the project name at the top, then click it again in order to switch into another project. Is there an easier and quicker way that I do not know of? Thanks.

Edit: Additionally, is there a way that I can switch between request collections within a single project via keyboard shortcut? Currently I click on the project name at the top left corner to switch to another request collection. There seems to be no mention of a keyboard shortcut at https://docs.insomnia.rest/insomnia/request-collections (at the time of this writing) and nothing within the keyboard shortcut menu either.

I'm not sure whether this should warrant its own discussion thread or not so I put it here, but if it's better as its own thread I can do that too.

**Comment by filfreire** [comment]
Hi @Coteh, you could try the following:

- Press `Ctrl+P` (or `Cmd+P` if you are on a Mac)
- You should see a menu that shows requests and also collections and design docs
- Start writing the name of the collection you want and select it with arrow keys and press enter (you can also select with the mouse pointer)

Example video:

https://user-images.githubusercontent.com/11976836/161237980-3270ef40-5a06-4fa0-90a7-a44df90637ac.mp4

**Reply by Coteh** [reply]
That's perfect, thanks! For some reason I didn't notice this earlier, I think if i have a lot of requests within a collection they get moved to the bottom of the list - sometimes out of view.

Is it possible to switch to a different project as well using this context menu?
